services/fluxmonitor.go

```diff
@@ -39,6 +40,8 @@ type FluxMonitorConfig struct {
 
 func ParseFMSpec(jsonSpec json.RawMessage, runtimeConfig store.RuntimeConfig) (FluxMonitorConfig, error) {
 	var fmConfig FluxMonitorConfig
+	var fmSpecErrors []string
+	var err error
```

Comment:
We can declare the return vars in the function signature so we don't have to declare them in the beginning of the function.

This also means that we can just `return` instead of returning with the return variables (`return fmConfig err`)

```suggestion
func ParseFMSpec(jsonSpec json.RawMessage, runtimeConfig store.RuntimeConfig) (fmConfig FluxMonitorConfig, err error) {
	var fmSpecErrors []string
```

Reply:
Nice!